src/charts/ScenarioManagerTreeList/ScenarioManagerTreeList.js

```diff
@@ -1,14 +1,25 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
 
-import React, { useEffect, useMemo, useRef, useState } from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import PropTypes from 'prop-types';
-import { TextField, Typography } from '@material-ui/core';
+import clsx from 'clsx';
+import { Button, Paper, TextField, Tooltip } from '@material-ui/core';
+import { UnfoldMore as UnfoldMoreIcon, UnfoldLess as UnfoldLessIcon } from '@material-ui/icons';
 import '@nosferatu500/react-sortable-tree/style.css';
 import SortableTree from '@nosferatu500/react-sortable-tree';
 import { ScenarioUtils } from '@cosmotech/core';
 import useStyles from './style';
 import { ScenarioNode } from '../../cards';
+import { SHRUNK_NODE_HEIGHT, EXPANDED_NODE_HEIGHT } from '../../cards/ScenarioNode/constants';
+
+const initNodesDict = (scenarios) => {
+  const nodesDict = {};
+  scenarios.forEach((scenario) => {
+    nodesDict[scenario.id] = false;
+  });
```

Comment:
In this case, shouln't we use `map` insteed of `forEach` ?
`const nodesDict = scenario.map((scenario) => {
    const dict = 
        return {[senario.id]: false};
}`

Reply:
`scenario.map(...)` returns an array, but we need a dict